### PR TITLE
Add allow and deny-insecure server arguments

### PR DESCRIPTION
**Pull request by nacfe**
## Change list

- Add flag "allow-insecure" in GeneralServerFlag.
- Add flag "deny-insecure" in GeneralServerFlag.
 
## Types of changes

- [x] No changes in production code.
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/main/java/io/appium/java_client/service/local/flags/GeneralServerFlag.java [hunk 1]
```diff
@@ -119,7 +119,13 @@
...
      * Enables NodeJS memory dumps collection feature.
      */
-    ENABLE_HEAP_DUMP("--enable-heapdump");
+    ENABLE_HEAP_DUMP("--enable-heapdump"),
+    /**
+     * Allow a list of features which are considered insecure and must be turned on explicitly by system administrators.
+     * Default: []
+     * Sample: --allow-insecure=foo,bar
+     */
+    ALLOW_INSECURE("--allow-insecure");
```
Comment (mykola-mokhnach) [inline src/main/java/io/appium/java_client/service/local/flags/GeneralServerFlag.java]: do we also have a constant for `--deny-insecure`?

Comment (nacfe) [reply src/main/java/io/appium/java_client/service/local/flags/GeneralServerFlag.java]: No. I added it now and changed this PR description.

**Comment by mykola-mokhnach** [comment]
Could you please sign the CLA?

**Comment by nacfe** [comment]
> Could you please sign the CLA?

I already did, but for some reason it says I didn't and doesn't allow me to edit the fields...

**Comment by mykola-mokhnach** [comment]
You need to make sure the email address of your local git account is the same that has been used to sign the CLA. After the address is fixed try to rebase the PR or create a new one if rebase does not help.

**Comment by nacfe** [comment]
> You need to make sure the email address of your local git account is the same that has been used to sign the CLA. After the address is fixed try to rebase the PR or create a new one if rebase does not help.

The email field was already filled with the same email I use on github when I entered the page for the first time. Maybe the fact that I had my email as private and not public made it fail?

**Comment by mykola-mokhnach** [comment]
Does `git config --list` show this email?

**Pull request closed without merging**